packages/reakit/src/Provider/Provider.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import {
+  Provider as ConstateProvider,
+  ProviderProps as ConstateProps
+} from "constate";
+import { ThemeProvider, ThemeProviderProps } from "../styled";
+
+interface ComponentProps
```

Comment:
Let's call it `ProviderProps` and export it the same way we're doing on https://github.com/reakit/reakit/pull/220#issuecomment-419767826.